chia/plotters/plotters.py

```diff
@@ -0,0 +1,359 @@
+            parser.add_argument(
+                "-t",
+                "--tmpdir",
+                type=str,
+                help="Temporary directory 1.",
+                default=str(root_path) + "/",
+            )
+        if option is Options.TMP_DIR2:
+            parser.add_argument(
+                "-2",
+                "--tmpdir2",
+                type=str,
+                help="Temporary directory 2.",
+                default=str(root_path) + "/",
+            )
+        if option is Options.FINAL_DIR:
+            parser.add_argument(
+                "-d",
+                "--finaldir",
+                type=str,
+                help="Final directory.",
+                default=str(root_path) + "/",
+            )
+        if option is Options.FILENAME:
+            parser.add_argument(
+                "--filename",
+                type=str,
+                help="Plot filename.",
+                default="plot.dat",
+            )
+        if option is Options.BUFF:
+            parser.add_argument(
+                "-b",
+                "--buffer",
+                type=int,
+                help="Size of the buffer, in MB.",
+                default=0,
+            )
+        r_default = 0 if name == "chiapos" else 4
+        if option is Options.NUM_THREADS:
+            parser.add_argument(
+                "-r",
+                "--threads",
+                type=int,
+                help="Num threads.",
+                default=r_default,
+            )
+        if option is Options.NOBITFIELD:
+            parser.add_argument(
+                "-e",
+                "--nobitfield",
+                type=bool,
+                help="Disable bitfield.",
+                default=False,
+            )
+        if option is Options.MEMO:
+            parser.add_argument(
+                "-m",
+                "--memo",
+                type=binascii.unhexlify,
+                help="Memo variable.",
+            )
+        if option is Options.ID:
+            parser.add_argument(
+                "-i",
+                "--id",
+                type=binascii.unhexlify,
+                help="Plot id",
+            )
+        if option is Options.PLOT_COUNT:
+            parser.add_argument(
+                "-n",
+                "--count",
+                type=int,
+                help="Number of plots to create (default = 1)",
+                default=1,
+            )
+        if option is Options.MADMAX_NUM_BUCKETS_PHRASE3:
+            parser.add_argument(
+                "-v",
+                "--buckets3",
+                type=int,
+                help="Number of buckets for phase 3+4 (default = 256)",
+                default=256,
+            )
+        if option is Options.MADMAX_WAITFORCOPY:
+            parser.add_argument(
+                "-w",
+                "--waitforcopy",
+                type=bool,
+                help="Wait for copy to start next plot",
+                default=True,
```

Comment:
```suggestion
                action="store_true",
                help="Wait for copy to start next plot",
                default=False,

Reply:
For the boolean options, the parser is currently expecting an argument. To keep the CLI similar to `chia plots create`, I'm thinking we can keep the boolean options standalone without requiring a 'true' or 'false' value (actually it's not clear what values are allowed, yes/no, y/n, true/false, 1/0, capitalized or not?)

In the case of `--waitforcopy` I think the default should be False (which would indicate that --waitforcopy wasn't specified in the command line). Personally I like having this option enabled, so I would always set it but perhaps I'm in the minority.